core/hw/pvr/ta_ctx.h

```diff
@@ -1,7 +1,30 @@
 #pragma once
+#include <malloc.h>
+#include <stdlib.h>
 #include "ta.h"
 #include "pvr_regs.h"
 
+// helper for 32 byte aligned memory allocation
+inline void* aligned_malloc(size_t size, size_t align)
+{
+    	void *result;
+    	#ifdef WIN32 
```

Comment:
Can you use `#if HOST_OS == OS_WINDOWS` instead?

Reply:
It is done.  I don't have a windows system to test it on though.  Do the automated checks (appveyor/travis-ci) cover this kind of thing?